nucypher/blockchain/eth/agents.py

```diff
@@ -167,21 +167,24 @@ def get_allowance(self, owner: ChecksumAddress, spender: ChecksumAddress) -> NuN
     @contract_api(TRANSACTION)
     def increase_allowance(self,
                            sender_address: ChecksumAddress,
-                           target_address: ChecksumAddress,
+                           spender_address: ChecksumAddress,
                            increase: NuNits
                            ) -> TxReceipt:
-        contract_function: ContractFunction = self.contract.functions.increaseAllowance(target_address, increase)
-        receipt: TxReceipt = self.blockchain.send_transaction(contract_function=contract_function, sender_address=sender_address)
+        """Increase the allowance of a spender address funded by a sender address"""
+        contract_function: ContractFunction = self.contract.functions.increaseAllowance(spender_address, increase)
+        receipt: TxReceipt = self.blockchain.send_transaction(contract_function=contract_function,
+                                                              sender_address=sender_address)
         return receipt
 
     @contract_api(TRANSACTION)
-    def approve_transfer(self, amount: NuNits,
-                         target_address: ChecksumAddress,
+    def approve_transfer(self,
+                         amount: NuNits,
+                         spender_address: ChecksumAddress,
                          sender_address: ChecksumAddress
                          ) -> TxReceipt:
-        """Approve the transfer of tokens from the sender address to the target address."""
+        """Approve the spender address to transfer an amount of tokens on behalf of the sender address"""
```

Comment:
...eh?  Spender v. Sender is confusing here IMO.

Reply:
I understand the potential confusion, but not sure how to proceed here. "spender" is the term used by the ERC20 implementation we use, by OpenZeppelin. And "sender" is the term we use in all the agents' methods.